taxi/models.py

```diff
@@ -15,7 +18,13 @@ def __str__(self):
 
 
 class Driver(AbstractUser):
-    license_number = models.CharField(max_length=255, unique=True)
+    license_number = models.CharField(max_length=8, unique=True)
+
+    def clean_number(self):
+        if not re.match(r"^[A-Z]{3}\d{5}$", self.license_number):
+            raise ValidationError(
+                "License number must consist of 3 "
+                "uppercase letters followed by 5 digits.")
```

Comment:
The method `clean_number` should be renamed to `clean_license_number` to ensure it is automatically called during the model's full_clean process. This is crucial for proper validation of the `license_number` field.